test/quicer_SUITE.erl

```diff
@@ -1177,6 +1180,48 @@ tc_stream_controlling_process(Config) ->
       ct:fail("timeout")
   end.
 
+%% @doc Check that old owner down will not shutdown the stream
+tc_stream_controlling_process_demon(Config) ->
+  Port = select_port(),
+  Owner = self(),
+  {SPid, Ref} = spawn_monitor(fun() -> echo_server(Owner, Config, Port) end),
+  receive
+    listener_ready ->
+      {ok, Conn} = quicer:connect("localhost", Port, default_conn_opts(), 5000),
+      Parent = self(),
+      {Old, MonRef} = spawn_monitor(
+                             fun() ->
+                                 {ok, Stm} = quicer:start_stream(Conn, [{active, false}]),
+                                 Res = quicer:controlling_process(Stm, Parent),
+                                 exit({Res, Stm})
+                             end),
+      receive
+        {'DOWN', MonRef, process, NewOwner, {Res, Stm}} ->
+          ct:pal("Set controlling_process res: ~p", [Res])
+      end,
+      ok = quicer:setopt(Stm, active, true),
+      {ok, _Len} = quicer:send(Stm, <<"owner_changed">>),
+      receive
+        {quic, <<"owner_changed">>, Stm, _} ->
+          ok
+      end,
+      %% Set controlling_process again
+      {NewOwner2, MonRef2} = spawn_monitor(fun() -> receive stop -> ok end
+                                           end),
+      %?assertEqual({error, owner_dead}, quicer:controlling_process(Stm, NewOwner)),
```

Comment:
This cause issue: set owner to dead pid will return error but makes the stream has no owner.